README.md

```diff
@@ -1 +1,9 @@
-# ClassicPress-Migration-Plugin
\ No newline at end of file
+# Switch to ClassicPress from WordPress
+
+Switch to ClassicPress from WordPress and say Goodbye to Gutenberg. (ClassicPress Migration Plugin)
```

Comment:
Please remove "and say Goodbye to Gutenberg"

Reply:
Ok to remove "and say Goodbye to Gutenberg"